server/data/constants.ts

```diff
@@ -34,6 +34,8 @@ export const roleApproveReject = 'INCIDENT_REPORTS__APPROVE' as const
  */
 export const rolePecs = 'INCIDENT_REPORTS__PECS' as const
 
+// TODO: will management reporting need a separate role?
```

Comment:
This is a good question, my feeling is no, as each report is controlled by an existing role and the caseloads.  We might need something specific but I think users can see reports on the prisons they working in

Reply:
is there any report that’s open to all ppl?